notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<p>Split this into two cells. Imports and checking GPU. Incase someone wants to debug the GPU, they don't need to import each time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Removed GPU check based on Benoit's comment. Thanks.